test/cypress/e2e/fixtures/texts.json

```diff
@@ -17,7 +17,7 @@
   "next": "Neste",
   "requiredField": "Feltet er påkrevd",
   "requiredFieldLastName": "Du må fylle ut nytt etternavn",
-  "requiredFieldDateFrom": "Du må fylle ut Dato for navneendring",
+  "requiredFieldDateFrom": "Du må fylle ut dato for navneendring",
```

Comment:
I updated `frontend-test` to use a lowercase label for this field. Just a minor annoyance I had, that I thought I'd fix when I had to deploy a new version of `frontend-test` on friday.